### PR TITLE
[10.0][FIX] shopinvader: Recompute also sale order lines

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
The idea is to recompute sale order lines fields too (as in some cases, values appear to be wrong).

The better should be to get a reentry loop on one2many fields and get all compute and store fields.

@lmignon An advice on this ?

File: shopinvader/models/sale.py [hunk 1]
```diff
@@ -209,6 +210,14 @@ class SaleOrderLine(models.Model):
         string="Shopinvader Variant",
     )
 
+    @api.multi
+    def shopinvader_recompute(self):
+        for line in self:
```
Comment (lmignon) [inline shopinvader/models/sale.py:215]: @rousseldenis This method should be on sale.order.line ????!!! Moreover, you break batch re computation! You must call recompute on all the records at once.

**Review by lmignon — CHANGES_REQUESTED** [review]

File: shopinvader/models/sale.py (hunk 1 above)
Comment (rousseldenis) [reply shopinvader/models/sale.py:215]: It's on sale order line (the models are mixed...)

Ok for recompute on recordset

**Review by Cedric-Pigeon — APPROVED** [review]
Code review, tests and used in prod

**Review by lmignon — APPROVED** [review]

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review)

**Pull request merged**